From 86649db88c859b7d46d7dc13333c73c9d50194ca Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Sun, 25 Dec 2016 13:01:52 +0200
Subject: [PATCH 1254/2241] dts: ap806: fix AXI monitor clocks

Change the clock name to hclk and add clock references for
south-bridge bound monitors (monitor traffic to south bridge)
- clocks are required to enabled profiling feature

Change-Id: Ic29437fab676b806064f77149881c7d870a3c05d
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/34960
Reviewed-by: Omri Itach <omrii@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-ap806.dtsi | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
index 4296b22..1d4c68e 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
@@ -225,7 +225,7 @@
 				compatible = "marvell,coresight-axim", "arm,primecell";
 				reg = <0x840000 0x1000>;
 				clocks = <&syscon 3>, <&syscon 5>;
-				clock-names = "apb_pclk", "dclk";
+				clock-names = "apb_pclk", "hclk";
 				bus-width = <40>;
 			};
 
@@ -233,23 +233,23 @@
 				compatible = "marvell,coresight-axim", "arm,primecell";
 				reg = <0x841000 0x1000>;
 				clocks = <&syscon 3>, <&syscon 5>;
-				clock-names = "apb_pclk", "dclk";
+				clock-names = "apb_pclk", "hclk";
 				bus-width = <40>;
 			};
 
 			axim-sb-rd@848000 {
 				compatible = "marvell,coresight-axim", "arm,primecell";
 				reg = <0x848000 0x1000>;
-				clocks = <&syscon 3>;
-				clock-names = "apb_pclk";
+				clocks = <&syscon 3>, <&syscon 7>;
+				clock-names = "apb_pclk", "hclk";
 				bus-width = <40>;
 			};
 
 			axim-sb-wr@849000 {
 				compatible = "marvell,coresight-axim", "arm,primecell";
 				reg = <0x849000 0x1000>;
-				clocks = <&syscon 3>;
-				clock-names = "apb_pclk";
+				clocks = <&syscon 3>, <&syscon 7>;
+				clock-names = "apb_pclk", "hclk";
 				bus-width = <40>;
 			};
 
-- 
2.7.4

